Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -155,5 +166,39 @@ private void OpenTab(string tabName)
                 }
             });
         }
+        
+        public void RefreshProfiles()
+        {
+            UpdateModel.Refresh();
+            CurrentState.AllProfiles = UpdateModel.AllProlfileNames;
+        }
+        
+        public void SetupWrapper()
+        {
+            var credentials = CoreApi.RetrieveAwsCredentials(CurrentState.SelectedProfile);
+            var client = new AmazonGameLiftClient(credentials.AccessKey, credentials.SecretKey);
+            GameLiftWrapper = new AmazonGameLiftWrapper(client);
+        }
+        
+        public void OpenS3Popup(string bucketName)
+        {
+            var popup = CreateInstance<GameLiftPluginBucketPopup>();
+            popup.Init(bucketName);
+            popup.OnConfirm += BootStrapPassthrough;
+            popup.ShowModalUtility();
+        }
+
+        private void BootStrapPassthrough(string bucketName)
```

Comment:
Can we call this `BootstrapAccount` for simpliclity?

Reply:
Same as above, please remove if this isn't relevant to this PR.

Reply:
But this is the correct place for the use of these. The Bucket interaction is created for this PR

Reply:
This is referenced within AwsCredentialsProfilePage.cs